code/Model/SiteTree.php

```diff
@@ -3248,25 +3256,24 @@ public function getPageIconURL()
      */
     public function classDescription()
     {
+        // Ensure base class has an appropriate description if not explicitly set,
+        // since we can't set that config for projects.
         $base = in_array(static::class, [Page::class, SiteTree::class]);
         if ($base) {
-            return $this->config()->get('base_description');
+            $baseDescription = static::config()->get('base_class_description');
```

Comment:
Move logic in this method to a new getClassDescription() method and update classDescription() to call $this->getClassDescription().

Do the same with i18n_classDescription() /  getI18nClassDescipcription()

Reply:
Conversation about this will be held in https://github.com/silverstripe/silverstripe-framework/pull/11461#discussion_r1839344166